### PR TITLE
Fixed nms for 1.16.4

### PR DESCRIPTION
**Pull request by W4545**

### CODE REVIEW

File: Spigot/src/main/java/net/notfab/hubbasics/spigot/nms/CraftBukkitVersion.java [hunk 1]
```diff
@@ -40,7 +40,9 @@
 
     v1_16_X("1.16.x", 1.16, "v1_16_R1", "v1_16_R2"),
```
Comment (Fabricio20) [inline Spigot/src/main/java/net/notfab/hubbasics/spigot/nms/CraftBukkitVersion.java:41]: Please update this entry with all NMS versions.

`v1_16_X("1.16.x", 1.16, "v1_16_R1", "v1_16_R2", "v1_16_R3")`

**Review by Fabricio20 — CHANGES_REQUESTED** [review]
Hey there @W4545 , thanks for the contribution! Just one small change for me to merge this if you can. The `.X` version needs to have all NMS names for the sake of grouping.

File: Spigot/src/main/java/net/notfab/hubbasics/spigot/nms/CraftBukkitVersion.java (hunk 1 above)
Comment (W4545) [reply Spigot/src/main/java/net/notfab/hubbasics/spigot/nms/CraftBukkitVersion.java]: I've modified the requested line

**Review by Fabricio20 — APPROVED** [review]
This will do it! Thank you so much for the contribution! I'll merge this in a few, setting up actions build.

**Pull request merged**